tests/archivers/test_archiver_base.py

```diff
@@ -0,0 +1,22 @@
+from auto_archiver.core import Metadata
+
+class TestArchiverBase(object):
+
+    archiver_class = None
+    config = None
+
+    def setUp(self):
+        assert self.archiver_class is not None, "self.archiver_class must be set on the subclass"
+        assert self.config is not None, "self.config must be a dict set on the subclass"
+        self.archiver = self.archiver_class(self.config)
+    
+    def create_item(self, url, **kwargs):
+        item = Metadata().set_url(url)
+        for key, value in kwargs.items():
+            item.set(key, value)
+        return item
+    
+    def assertValidResponseMetadata(self, test_response, title, timestamp):
+        self.assertTrue(test_response.is_success())
+        self.assertEqual(title, test_response.get_title())
+        self.assertTrue(timestamp, test_response.get("timestamp"))
```

Comment:
we may need to updated this based if some archivers don't provide these, but let's keep it for now and adapt as needed later.

Reply:
Noted, yes - I though this could be a good 'template' and we can adjust as we add more tests